edq/cli/config/set.py

```diff
@@ -0,0 +1,81 @@
+"""
+Set a configuration option.
+"""
+
+import argparse
+import sys
+import typing
+
+import edq.core.argparser
+import edq.core.config
+
+def run_cli(args: argparse.Namespace) -> int:
+    """ Run the CLI. """
+
+    config: typing.Dict[str, str] = {}
+    for config_option in args.config_to_set:
+        (key, value) = edq.core.config._parse_cli_config_option(config_option)
+        config[key] = value
+
+    # Defaults to the local configuration if no configuration type is specified.
+    if (not (args.write_local or args.write_global or (args.write_file_path is not None))):
+        args.write_local = True
+
+    if (args.write_local):
+        local_config_path = args._config_params[edq.core.config.LOCAL_CONFIG_PATH_KEY]
+        if (local_config_path is None):
+            local_config_path = args._config_params[edq.core.config.FILENAME_KEY]
+        edq.core.config.write_config_to_file(local_config_path, config)
+    elif (args.write_global):
+        global_config_path = args._config_params[edq.core.config.GLOBAL_CONFIG_PATH_KEY]
+        edq.core.config.write_config_to_file(global_config_path, config)
+    elif (args.write_file_path is not None):
+        edq.core.config.write_config_to_file(args.write_file_path, config)
+
```

Comment:
Add an `else` and raise an exception.
We usually do this with things like this and enums to protect us in the future if someone adds a new possible value.
Like if we decided to add system-level config, then an else would protect us.